frontend/src/pages/Projects/Project/index.tsx

```diff
@@ -21,15 +22,38 @@ export const Project = () => {
       {true === query.isLoading ? (
         <div>Loading</div>
       ) : (
-        Object.entries(query.data?.data).map(([key, value]) => {
-          return (
-            <div key={key}>
-              <br />
-              <TextField disabled label={key} defaultValue={value} />
-              <br />
-            </div>
-          );
-        })
+        <>
+          <GDX_Accordion sectionTitle="Project Registration">
+            {/* Added a slice just to show a few projects for now. */}
+            {Object.entries(query.data?.data)
+              .slice(1, 4)
+              .map(([key, value]) => {
+                return (
+                  <div key={key}>
+                    <br />
+                    <TextField disabled label={key} defaultValue={value} />
+                    <br />
+                  </div>
+                );
+              })}
+          </GDX_Accordion>
+          <br />
```

Comment:
is this best way to create space. with a br tag?

Reply:
Made it better in revision, good comment.  I was going quick for MVP :)